services/ip_cam_trendnet_tv_ip100/ip_cam_trendnet_tv_ip100_service.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""Honeycomb IP Cam TRENDnet TV-IP100 Service."""
+from __future__ import unicode_literals
+
+import os
+
+from six.moves.urllib import parse as urlparse
+from six.moves.BaseHTTPServer import HTTPServer
+from six.moves.SimpleHTTPServer import SimpleHTTPRequestHandler
+from six.moves.socketserver import ThreadingMixIn
+
+import requests
+
+from honeycomb.servicemanager.base_service import ServerCustomService
+
+DEFAULT_PORT = 80
+EVENT_TYPE_FIELD_NAME = "event_type"
+TRENDNET_ADMIN_ACCESS_EVENT = "trendnet_tv_ip100_admin_access"
+ORIGINATING_IP_FIELD_NAME = "originating_ip"
+ORIGINATING_PORT_FIELD_NAME = "originating_port"
+REQUEST_FIELD_NAME = "request"
+DEFAULT_SERVER_VERSION = "Camera Web Server/1.0"
+CAMERA_IMAGE_PATH = "/image.jpg"
+
+DEFAULT_CONTENT_TYPE = "image/jpeg"
+
+
+class ThreadingHTTPServer(ThreadingMixIn, HTTPServer):
```

Comment:
please split the server and service logic to two separate files so easier to comprehend 
`ip_cam_service.py` and `ip_cam_server.py`

Reply:
That's not our code, thats how it is in the simple_http service

Reply:
yes, simple_http is a basic example, but look at more complex services (pretty much any other service actually) and you'll see a separation between server and service

it just makes life easier for others reading your code